slam-libraries/viam-cartographer/src/io/file_handler_test.cc

```diff
@@ -76,9 +76,7 @@ BOOST_AUTO_TEST_CASE(RemoveFile_success) {
 
     // Remove a file
     int file_num = 1;
-    int success =
-        RemoveFile(tmp_dir.string() + "/data/" + data_files.at(file_num));
-    BOOST_TEST(success == file_num);
```

Comment:
As per my comment above, we were turning this function into returning void. Since we check the contents of the folder below to confirm the correct file was removed, this check here was not necessary as a failure below would catch it